### DISCUSSION TITLE
hidden attribute?

### DISCUSSION THREAD

**Discussion by simplymequeeny** [JedWatson/react-select#5302] [Q&A]
i looked into the documentation on how to dynamically show hide the dropdown list (select element) but could not find any.
is there a way to dynamically show / hide the dropdown list?

**Comment by manjushsh** [comment]
Isn't this done by default? 
Can you give example of 

>  how to dynamically show hide the drop down list

**Reply by simplymequeeny** [reply]
like the hidden attribute of HTML element
` <p hidden>This paragraph should be hidden.</p> `
in reactjs, it would be
` <p hidden={true|false}>This paragraph should be hidden.</p> `
